microsoft/testsuites/network/networksettings.py

```diff
@@ -667,7 +667,7 @@ def after_case(self, log: Logger, **kwargs: Any) -> None:
     def _check_msg_level_change_supported(self, node: Node) -> None:
         msg_level_symbols: Union[str, List[str]]
         uname_tool = node.tools[Uname]
-        kernel_version = uname_tool.get_linux_information().kernel_version
+        kernel_version = uname_tool.get_linux_information().kernel_version_raw
```

Comment:
How about other distros? Is there regression?

Reply:
I tested below images, no regression

```
        - openlogic centos 7_9 7.9.2022020700
        - openlogic centos-lvm 8-lvm 8.5.2022012000
        - debian debian-10 10 0.20220815.1108
        - debian debian-11 11 0.20220816.1109
        - oracle oracle-linux ol79 7.9.9
        - oracle oracle-linux ol86-lvm 8.6.3
        - microsoftcblmariner cbl-mariner cbl-mariner-2 2.20220804.01
        - microsoftcblmariner cbl-mariner cbl-mariner-1 1.20220805.01
        - redhat rhel 7_9 7.9.2022032201
        - redhat rhel 8_3 8.3.2022031401
        - redhat rhel 9_0 9.0.2022081813
        - suse sles-12-sp5 gen1 2022.07.14
        - suse sles-15-sp4 gen1 2022.06.21
        - suse opensuse-leap-15-4 gen1 2022.07.22
        - canonical ubuntuserver 16.04-lts 16.04.202109280
        - canonical ubuntuserver 18.04-lts 18.04.202207120
        - canonical 0001-com-ubuntu-server-focal 20_04-lts 20.04.202208100
        - canonical 0001-com-ubuntu-server-jammy 22_04-lts 22.04.202208100
        - openlogic centos 7_9-arm64 7.9.2022092602
        - debian debian-11 11-backports-arm64 0.20221121.1206
        - microsoftcblmariner cbl-mariner cbl-mariner-2-arm64 2.20221215.01
        - redhat rhel-arm64 8_6-arm64 8.6.2022060901
        - suse opensuse-leap-15-4-arm64 gen2 2022.11.04
        - suse sles-15-sp4-arm64 gen2 2022.11.04
        - canonical ubuntuServer 18_04-lts-arm64 18.04.202212090
        - canonical 0001-com-ubuntu-server-focal 20_04-lts-arm64 20.04.202212140
        - canonical 0001-com-ubuntu-server-jammy 22_04-lts-arm64 22.04.20221214
```